zmq-eio/test/test.ml

```diff
@@ -0,0 +1,141 @@
+open OUnit
+
+(** Sleep is horrible slow, or causing strange slowdowns *)
```

Comment:
This comment is quite unclear. Maybe worth a report upstream?

Reply:
Well. This was indeed not the case and the comment should be removed. It was before I discovered the bug in the existing lwt/async tests.

Reply:
I will remove the wrong comment